src/config.c

```diff
@@ -1077,9 +1073,12 @@ int config_write_template(const char *file, const struct config *cfg)
 			 "# httpreq_dns\t\t8.8.8.8\n"
 			 "# httpreq_hostname\tmyserver\n"
 			 "# httpreq_cert\t\tcert.pem\n"
-			 "# httpreq_key\t\tkey.pem\n"
-			 );
-
+			 "# httpreq_key\t\tkey.pem\n");
```

Comment:
unrelated change. The ");" was on the next line.

Reply:
Do we really care - that makes it the same as the other sections?